bitnami/keycloak/templates/tls-secret.yaml

```diff
@@ -39,5 +39,34 @@ data:
   tls.crt: {{ $cert.Cert | b64enc | quote }}
   tls.key: {{ $cert.Key | b64enc | quote }}
   ca.crt: {{ $ca.Cert | b64enc | quote }}
+---
+{{- end }}
 {{- end }}
+{{- if (include "keycloak.createTlsSecret" .) }}
+{{- $ca := genCA "keycloak-ca" 365 }}
```

Comment:
Shouldn't you include "alternative names"?

Reply:
Alternative Names are defined in line 63 and they are redefined for each Keycloak replica certificate.